resources/[system]/scoreboard-basic-theme/style.css

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600&display=swap');
```

Comment:
Remote dependencies are a no.

Reply:
I assume you'd like me to download the font instead?